lib/nitcorn/http_request.nit

```diff
@@ -82,7 +82,7 @@ class HttpRequest
 	fun int_arg(arg_name: String): nullable Int do
 		if not all_args.has_key(arg_name) then return null
 		var i = all_args[arg_name]
-		if not i.is_numeric then return null
+		if not i.is_int then return null
 		return i.to_i
```

Comment:
Personally, I’m not fan of these undocumented « features » (truncation here). For me, this goes against POLA. Maybe instead the documentation should be updated to clearly state that the value must be an integer?